app/models/parser/base.rb

```diff
@@ -69,7 +69,7 @@ def oembed_url(doc)
     attr_reader :unavailable_page
 
     # Implemented by subclasses
-    def parse_data_for_parser(doc, original_url, jsonld)
+    def parse_data_for_parser(doc, original_url, jsonld_array)
```

Comment:
Not at all, but I do have a question. This is what that part of the code looks like:

```
    def parse_data_for_parser(doc, original_url, jsonld_array)
      raise NotImplementedError.new("Parser subclasses must implement parse_data_for_parser")
    end
```

Should the `_` be added only to `jsonld_array`, should we add to all, should we use` *` ?

Reply:
Oh, I'm glad you asked; I didn't realize this alert was on the base parser -- as a result, this alert is kind of irrelevant since we don't use any of those variable names in this method. Instead, it's meant to define an interface for subclasses to implement.

Because of that, I think we should leave the variable names as-is. In this case the interface that we're defining (three variables with specific meanings) is important, so we wouldn't want to use the * since it hides what we want the subclasses to pass in.

Sorry for any confusion!